rllib/algorithms/algorithm.py

```diff
@@ -1672,15 +1673,14 @@ def _offline_eval_runner_remote(runner, iter):
 
             _round += 1
 
-            self.offline_eval_runner_group.foreach_runner_async(
-                func=functools.partial(
-                    _offline_eval_runner_remote,
-                    iter=algo_iteration,
-                ),
-            )
-            results = self.offline_eval_runner_group.fetch_ready_async_reqs(
-                return_obj_refs=False, timeout_seconds=0.01
```

Comment:
@simonsays1980 I'm not sure why the timeout was set to 0.01 here before.
Maybe I'm missing something but I'd think that it hinders execution by blocking for max 0.01s.